addons/mail/static/src/less/chat_window.less

```diff
@@ -47,14 +47,18 @@
             .o-text-overflow();
         }
 
-        .o_chat_window_close, .o_chat_window_expand {
-            color: white;
-            padding: 3px;
-            margin-left: 5px;
-            &:hover {
-                background-color: darken(@odoo-brand-primary, 10%);
+        .o_chat_buttons {
+            .o-flex-display();
```

Comment:
I think that what you want here is `.o-flex(0, 0, auto);` instead. Could you try ?